libraries/core-react/src/components/Menu/MenuList.tsx

```diff
@@ -26,7 +28,7 @@ const List = styled.ul`
   list-style: none;
   padding: 0;
   margin: 0;
-
+  ${spacingsTemplate(tokens.spacings)}
```

Comment:
`Menu` was missing top and bottom spacing. `Padding: 0` is needed to strip `ul` padding
![Screenshot 2021-06-22 at 12 49 16](https://user-images.githubusercontent.com/31915755/122913125-5bc32d80-d359-11eb-8245-15e4c61bddb7.png)